functional_tests.go

```diff
@@ -2170,43 +2171,12 @@ func testPutObjectWithChecksums() {
 			return
 		}
 
-		if err := r.Close(); err != nil {
-			logError(testName, function, args, startTime, "", "Object Close failed", err)
-			return
-		}
-		if err := r.Close(); err == nil {
-			logError(testName, function, args, startTime, "", "Object already closed, should respond with error", err)
-			return
-		}
-
-		args["range"] = "true"
-		err = gopts.SetRange(100, 1000)
-		if err != nil {
-			logError(testName, function, args, startTime, "", "SetRange failed", err)
-			return
-		}
-		r, err = c.GetObject(context.Background(), bucketName, objectName, gopts)
-		if err != nil {
-			logError(testName, function, args, startTime, "", "GetObject failed", err)
-			return
-		}
-
-		b, err = io.ReadAll(r)
+		// Discard the object
```

Comment:
Why did you delete all this?

Checking what checksum range requests return seems valuable.

Reply:
It was deleted during testing, please check again.